pms/__manifest__.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2019 Darío Lodeiros, Alexandre Díaz, Jose Luis Algara, Pablo Quesada
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "PMS (Property Management System)",
+    "summary": "A property management system",
+    "version": "14.0.2.37.0",
+    "development_status": "Beta",
+    "category": "Generic Modules/Property Management System",
+    "website": "https://github.com/OCA/pms",
+    "author": "Commit [Sun], Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": True,
+    "installable": True,
+    "depends": [
+        "base",
+        "base_automation",
+        "mail",
+        # "account_payment_return",
+        # "email_template_qweb",
+        "sale",
+        "multi_pms_properties",
+        "partner_firstname",
+        "partner_second_lastname",
+        "partner_contact_gender",
+        "partner_contact_birthdate",
+        "partner_contact_nationality",
+        "account_reconciliation_widget",
```

Comment:
Hi !
I have tested the PR and I detect an issue on this dependency, as the add-on `account_reconciliation_widget` has not been migrated to 16.0 : https://github.com/OCA/account-reconcile
Thks